src/main/webapp/js/user-page-loader.js

```diff
@@ -117,7 +117,13 @@ function fetchAboutMe() {
         indexVue.user_teachcate=info.teachCate;
         indexVue.user_level=info.skillLevel;
         indexVue.user_school=info.school;
-        document.getElementById('about-me-container').innerHTML=info.aboutMe;
+        document.getElementById('picc').src = info.pic;
```

Comment:
Hi Alison, there is no html element with id = "picc" or "url" in the user-page.html. This seems to be the problem.